web/src/components/__tests__/DatePicker.spec.tsx

```diff
@@ -17,19 +17,23 @@ describe('DatePicker', () => {
     const title = 'From Date'
     const date = DateTime.now()
 
+    const placeholderDate = DateTime.fromISO('2025-04-08')
+
     const { getByText, getByPlaceholderText } = renderCustomDatePicker({
       title,
       date,
       setDate,
       error: '',
+      placeholderDate,
     })
 
     expect(getByText(title)).toBeInTheDocument()
-    expect(getByPlaceholderText('01.01.1990')).toBeInTheDocument()
+    expect(getByPlaceholderText(placeholderDate.toFormat('dd.MM.yyyy'))).toBeInTheDocument()
```

Comment:
@steffenkleinle Is it ok to expect a variable like this or better to be hard coded like a string (for tests)?